src/test/kotlin/racing/domain/CarTest.kt

```diff
@@ -9,14 +10,20 @@ internal class CarTest {
     fun `자동차 움직임 테스트`() {
         val car = Car()
         car.tryMove(object : Movable { override fun isMovable(): Boolean = true })
```

Comment:
IntelliJ IDEA의 코드 자동 정렬 기능(⌥⌘L, Ctrl+Alt+L)을 사용하면 더 깔끔한 코드를 볼 수 있을 거예요.

```suggestion
        car.tryMove(object : Movable {
            override fun isMovable(): Boolean = true
        })
```